vic/drivers/image/src/vic_force.c

```diff
@@ -220,6 +321,17 @@ vic_force(void)
                                                 param.SNOW_MAX_SNOW_TEMP,
                                                 &(atmos[i].prec[j]), 1);
         }
+        // Validate vegcover
+        for (v = 0; v < options.NVEGTYPES; v++) {
+            vidx = veg_con_map[i].vidx[v];
+            if (vidx != -1) {
+                for (j = 0; j < NF; j++) {
+                    if (veg_hist[i][vidx].vegcover[j] < MIN_VEGCOVER) {
+                        veg_hist[i][vidx].vegcover[j] = MIN_VEGCOVER;
+                    }
+                }
+            }
+        }
```

Comment:
I think any time that we change a user defined input - even if there are good physical reasons to do so - we should not do so silently, because users should be aware that their inputs are modified. For example, if they are analyzing their results assuming a certain LAI value, then they need to be aware that this has been changed.

It may actually be nice to put all warning messages in the documentation where we can include a more complete explanation of why the warning is necessary (for example the explanation from @tbohn above). That way we can refer in the warning to the correct place in the documentation and keep the warning short. 

So as far as I am concerned as long as we warn I think this is fine.